ttnn/cpp/ttnn/operations/normalization/softmax/device/multi_core/softmax_op_multi_core.cpp

```diff
@@ -387,15 +400,37 @@ operation::ProgramWithCallbacks scale_mask_softmax_multi_core(
             uint32_t mask_offset = curr_row / Ht * Wt * Wt; // causal mask batch offset
             uint32_t mask_id = causal_mask ? (mask_curr_ht * Wt + mask_offset) : (curr_row / Ht * Wt); // causal mask start offset + causal mask batch offset
 
+            reader_kernel_args[0] = src_buffer_address;
+            reader_kernel_args[1] = block_size;
+            reader_kernel_args[2] = s.u;
+            reader_kernel_args[3] = num_tile_rows_per_core;
+            reader_kernel_args[4] = tile_offset;
+            reader_kernel_args[5] = Wt;
+            reader_kernel_args[6] = Ht;
+            reader_kernel_args[7] = mask_buffer_address;
+            reader_kernel_args[8] = curr_ht;
+            reader_kernel_args[9] = mask_id;
+            reader_kernel_args[10] = 0x3f803f80;
```

Comment:
This hardcoded value doesn't need to be updated if we initialize it to this instead of 0 on line 237

Reply:
Ok

Reply:
Still need to update line 237 to not set 0x3f803f80 instead of 0.

Reply:
Is it ok now?